php_remote_error_monitor.h

```diff
@@ -48,7 +48,13 @@
     #define PHP_REMOTE_ERROR_MONITOR_API
   #endif
 
-  #define REMOTE_ERROR_MONITOR_E_ALL (E_ALL & ~E_DEPRECATED & ~E_NOTICE & ~E_WARNING | E_STRICT)
+  /* Saving this specification, as we may want to return to it when our
+   * error reporting mechanism isn't causing platform stability issues.
+
+    #define REMOTE_ERROR_MONITOR_E_ALL (E_ALL & ~E_DEPRECATED & ~E_NOTICE & ~E_WARNING | E_STRICT)
+   */
```

Comment:
-1 on "saving this in a comment"— that's what version control is for.

Reply:
alright